frontend/src/components/browse-sightings-page/BrowseSightings.tsx

```diff
@@ -1,3 +1,27 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { getSightings, Sighting } from "../../clients/apiClient";
+import { SightingCard } from "./SightingCard";
+import "./BrowseSightings.scss";
 
-export const BrowseSightings: React.FunctionComponent = () => <></>;
+export const BrowseSightings: React.FC = () => {
+  const [sightings, setSightings] = useState<Sighting[]>();
+
+  useEffect(() => {
+    getSightings().then(setSightings);
+  }, []);
+
+  if (sightings === undefined) {
+    return <p>Loading</p>;
+  }
+
+  return (
+    <div className="feed">
+      <h1 className="title">Reported Sightings</h1>
+      <div>
+        {sightings.map((sighting, index) => (
+          <SightingCard sighting={sighting} key={index} />
+        ))}
+      </div>
+    </div>
```

Comment:
doesn't react need a <div> right after return(  ? 
 the <div> is also there to contain / align the page.

Reply:
> doesn't react need a `<div>` right after `return(` ?
Not quite - the rule is that React elements can only have one element at the top-level, so you need to group them together using _something_ (that's what the JSX Fragment, `<> </>`, is for!)

So:
```
return <div>
  <h1>Hello</h1>
  <p>Hello!</p>
</div>
```
is valid, and
```
return <>
  <h1>Hello</h1>
  <p>Hello!</p>
</>
```
is valid, but
```
return (
  <h1>Hello</h1>
  <p>Hello!</p>
);
```
is _not_ valid, because it has more than one element at the top level.

> what would make sense is to take the `className="feed"` ( now called page-content) and put in the app.tsx to apply on all pages.

I'm happy for you to apply this class to the `<main>` in `App.tsx`, and style it in the CSS with the `main.page-content` selector, yes! 😊